src/components/app/tour-list/TourList.tsx

```diff
@@ -4,18 +4,44 @@ import TourListActions from './TourListActions'
 import React from 'react'
 import dayjs from 'dayjs'
 import { dateTimeFormat } from '../../../utils/constants'
+import { OfflineBolt } from '@mui/icons-material'
+import { TourStatusType } from '../../../enums/tour-status-type'
+import useOnlineStatus from '../../../hooks/use-online-status'
+import { Chip } from '@mui/material'
 
 type TourListProps = {
   rows: Tour[],
   loading: boolean,
 }
 
-function getActions (params: GridValueGetterParams<Tour, Tour>): JSX.Element {
+const getActions = (params: GridValueGetterParams<Tour, Tour>): JSX.Element => {
   return <TourListActions id={params.row.id} />
 }
 
+const getName = (params: GridValueGetterParams<Tour, Tour>): JSX.Element => {
+  const isOnline = useOnlineStatus()
+  const offlineCreatedChip: JSX.Element = <Chip size="small" sx={{ ml: 1 }} label="Complete to sync" color="primary"/>
+  const offlineBolt: JSX.Element = <span title="This tour is not synchronized with the database."><OfflineBolt sx={{ mr: 1 }}/></span>
```

Comment:
I think we could leave out this type assertion, because it's clear (https://docs.gipfeli.io/docs/guidelines/conventions#types)

(However, I also had to check several times, because I am not sure we have our style guide fleshed out correctly yet 😆 )